### PR TITLE
add contribution guide

### PR DESCRIPTION
**Pull request by rpitonak**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+### How to contribute code to release-bot
+
+1. Create a fork of the `release-bot` repository.
+2. Create a new branch just for the bug/feature you are working on.
+
+   - If you want to work on multiple bugs/features, you can use branches to keep them separate, so that you can submit a separate Pull Request for each one.
+
+3. Once you have completed your work, create a Pull Request, ensuring that it meets the requirements listed below.
+
+### Requirements for Pull Requests
```
Comment (jpopelka) [inline CONTRIBUTING.md]: I'd probably move this and 'How to contribute code to release-bot' before 'Testing'.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  trying to convince the person to use the project and that the changelog
+  should help with that.
+
+
+### Testing
+
+For testing, we are using [pytest](https://docs.pytest.org/en/latest/) framework. Tests are stored in the [tests](/tests) directory. We recommend to run tests inside docker container using:
+```
+# if TEST_TARGET is empty whole test suite is executed
+make test-in-container TEST_TARGET=<PATH>
```
Comment (TomasTomecek) [inline CONTRIBUTING.md]: Can we reword this a bit? let's do this example without TEST_TARGET and do the other one with it? And say that you can also run specific tests.

File: README.md [hunk 3]
```diff
@@ -149,16 +149,3 @@ new version of this image is pushed to docker hub.
 You can change it by uncommenting lines with `#importPolicy:`
 and `#scheduled: true` in [openshift-template.yml](openshift-template.yml).
 Then the image will be pulled on a new release.
-
-
-# Contributing
```
Comment (TomasTomecek) [inline README.md]: Maybe add a short sentence here that we have contribution guide?

**Comment by rpitonak** [comment]
Updated, PTAL.

**Pull request merged**